src/main/java/org/breedinginsight/brapps/importer/model/base/ObservationUnit.java

```diff
@@ -50,10 +52,16 @@
 
     @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
     @ImportFieldMetadata(id="observationLevel", name="Observation Level",
-            description = "The type of the observation unit. Example: Plot, Plant.")
+            description = "The type of the observation unit. Allowable options: plot, plant.")
     @ImportMappingRequired
     private String observationLevel;
 
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="observationLevelCode", name="Observation Level Code",
+            description = "An ID code for this level tag. Example: plot number for a plot observation level.")
```

Comment:
Maybe something saying that plot numbers must be unique within a study?

Reply:
ya, I'll add that

Reply:
pushed changes